src/openmc_heat_driver.cpp

```diff
@@ -52,20 +53,28 @@ void OpenmcHeatDriver::init_mappings()
   const auto& z = heat_driver_->z_;
 
   std::unordered_map<CellInstance, int> tracked;
+
+  // index for rings in solid
   int ring_index = 0;
+
+  // index for elements in fluid
+  int elem_index = 0;
```

Comment:
I would move the declaration further down so that it's closer to where it's actually used.